app/redux/ducks/classify.js

```diff
@@ -51,6 +51,7 @@ function createNewClassification(project, workflow, subject, goldStandardMode, l
   // Delete the metadata key because we don't want volunteers to see it.
   subject.update({ 'metadata.intervention': undefined });
   const newMetadata = {
+    _inProgress: false,
```

Comment:
The underscore here ensures that the Panoptes JS Client will automatically remove this property when it saves the classification to the API.
https://github.com/zooniverse/panoptes-javascript-client/blob/721b696dccecf86a35614244ab29d19d1a62c3b7/lib/json-api-client/model.js#L17-L38